go.mod

```diff
@@ -5,19 +5,23 @@ require (
 	github.com/getkin/kin-openapi v0.115.0
 	github.com/gin-gonic/gin v1.9.0
 	github.com/go-chi/chi/v5 v5.0.8
+	github.com/gofiber/adaptor/v2 v2.2.0
```

Comment:
The `adaptor` middleware now is part of fiber stdlib: https://github.com/gofiber/fiber/tree/77c1b4868a686254a839fbbb88d7f46cbadb055a/middleware/adaptor

Reply:
I dont think this has been versioned yet

Reply:
fiber v2.46.0 just dropped 🥳